server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/routes/HealthCheckRoutes.java

```diff
@@ -183,7 +184,7 @@ private void logFailedCheck(Result result) {
 
     private ImmutableList<Result> executeHealthChecks() {
         return healthChecks.stream()
-            .map(healthCheck -> healthCheck.check().block())
+            .map(healthCheck -> Mono.from(healthCheck.check()).block())
```

Comment:
use `flatMap`

Reply:
What's the point of using flatMap here? It is not a reactive stream and we are transforming 1 to 1 ?

Reply:
any reason you don't start with a `Flux.fromIteratble(healthChecks)`?
It would avoid the `block` calls

Reply:
No particular reason. Because the original code was blocking so i don't put a lot of thought in changing it.

Reply:
> What's the point of using flatMap here? It is not a reactive stream and we are transforming 1 to 1 ?

We need to perform a synchronization to get it, let Reactor do it it's way and avoid to use `block` into reactive code